pkg/recommendation/ingress_tls_secret_valid.go

```diff
@@ -0,0 +1,90 @@
+package recommendation
```

Comment:
I assume that the code logic was not changed, right? there is no need to "recheck" it?

Reply:
Nope, I just simplified / slightly improved the code overall without changing the way how it works.

For Pod Labels Set I got rid of one snippet which wasn't necessary.

I tested all recommendations manually + implemented unit tests as well.